### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by EvansSnave**
Created desktop version. In stule.css added media querys for the responsive design. The navbar is fixed when in desktop size. Text and images change when switching from mobile version
In index.html changed some containers order and added classes to elements from previous sections.

### CODE REVIEW

**Review by Favourezeugwa — COMMENTED** [review]
Hi @EvansSnave,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌❌❌

You have submitted a project, that does not have linters passing or showing the green tick. Kindly fix the errors and re-submit.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by EvansSnave** [comment]
Hi. I am still having issues with the linters. I have tried everything in internet and even had a mentor hemping me for like two hours but still my linter errors wont dissapear. that is why i submit this code with those two linter error.

**Comment by Favourezeugwa** [comment]
I am really sorry because I wouldn't be able to review or approve a PR
that doesn't have linters passing. What errors were you facing? Is it
something in the CSS?

**Comment by EvansSnave** [comment]
Yes I am having two linter errors in my style.css file where it says I have two duplicate selectors in specific lines, but I already checked several times and there are not duplicated seslector anywhere. Already installed and uninstalled my linter and reseted its configurations with a mentor's help but still won't work

**Comment by Favourezeugwa** [comment]
Have you tried adding “ —fix “at the end of the stylint linters command?

**Comment by EvansSnave** [comment]
Yes multiple times, with and without it. I even reseted my computer as my mentor told me.

**Comment by EvansSnave** [comment]
Hey I thin I finally solved it somehow. Can you review it now please?

**Comment by EvansSnave** [comment]
@Favourezeugwa I cant request another request because it says review pending.

File: Style.css [hunk 1]
```diff
@@ -462,3 +451,197 @@ body {
...
+.buttonMenu { display: none; }
+
+.mobile2 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.buttonC:hover 
+{
```
Comment (didierganthier) [inline Style.css]: I believe the issue is here 👇🏽 

Screenshot 2023-05-07 at 01 15 27

Should be fixed if you replace these two lines by this :

```
.buttonC:hover {
```

**Review by didierganthier — COMMENTED** [review]
Hi @EvansSnave ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project with linters check failing. ❌ 

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Comment by didierganthier** [comment]
@EvansSnave I just noticed that they are fixed now. I'm reviewing. 👍🏽

File: Index.html [hunk 2]
```diff
@@ -13,13 +13,19 @@
 <body>
   <header class="navbar">
     <a href="index.html" class="myLogo">My logo</a>
```
Comment (didierganthier) [inline Index.html:15]: [OPTIONAL] Make sure to make some adjustments also on the mobile version 👀 this is what I get for the Logo on Mobile 👇🏽 

Screenshot 2023-05-07 at 01 35 42

File: Index.html [hunk 3]
```diff
@@ -30,109 +36,156 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+              <li class="iconw"><img class="image" src="Assests/Ellipse 1htmlLogo.svg">HTML</li>
+              <li class="iconw"><img class="image" src="Assests/Ellipse 1css.svg">CSS</li>
+            </ul>
+          </li>
+        </ul>
         <li class="frameworks languages">Frameworks<img class="droplet side" src="Assests/droplet.png"></li>
         <li class="skills languages">Skills<img class="droplet sideTwo" src="Assests/droplet.png"></li>
-      </ul>
+      </div>
+    </div>
```
Comment (didierganthier) [inline Index.html:188]: [OPTIONAL] This section is overflowing with the contact one 🤔 now this issue is only present when I test directly on a phone size of `375` px but it's not there when I resize responsively, take a look 👇🏽 

Screenshot 2023-05-07 at 01 36 21

**Review by didierganthier — APPROVED** [review]
Hi @EvansSnave ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Used `media-queries` ✔️ 
- Implemented Figma Design ✔️ 
- Correct GitHub Flow Used ✔️  

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Comment by EvansSnave** [comment]
@didierganthier Thank you so much

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @EvansSnave ,:wave: :wave: 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 🛑

You have submitted a project with a pull request already merged. As you have a working link, I suggest you create an issue and add that link in your issue.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**